Edit task

volteryanec opened the pull request.

volteryanec committed:
Update index.js

volteryanec committed:
Поправил Edit Task

nikitamax commented on index.js:
```diff
@@ -85,7 +86,51 @@ function deleteCurentTask(event) {
 }
+function changeClassLi() {
+  li = event.target.parentElement.parentElement;
+  li.className = li.className == "completed" ? "completed editing" : "editing";
```
можно короче  - li.className = li.className ? li.className + " editing" : "editing";

nikitamax commented on index.js:
```diff
@@ -85,7 +86,51 @@ function deleteCurentTask(event) {
+function inputEditBlur() {
+  li = event.target.parentElement;
+  if (event.target.value == "") {
+    tasksList.forEach(function(elem, index) {
```
повторяющийся код нужно вынести в отдельную функцию

volteryanec committed:
Made changes to edit-task

li.className = li.className ? li.className + " editing" : "editing"; С этим у меня пока туго никак не могу привыкнуть.
